src/components/layout/KeyboardFixedOpenLayout.js

```diff
@@ -32,17 +31,16 @@ export default function KeyboardFixedOpenLayout({
   position = android ? 'relative' : 'absolute',
   ...props
 }) {
-  const insets = useSafeArea();
+  const insets = useSafeAreaInsets();
   const { height: screenHeight } = useDimensions();
   const keyboardHeight = useKeyboardHeight({
     keyboardType,
   });
   const ref = useRef();
 
   const containerHeight = screenHeight - keyboardHeight - additionalPadding;
-  ios &&
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    useEffect(() => ref.current.animateNextTransition(), [containerHeight]);
+
+  useEffect(() => ref.current.animateNextTransition(), [containerHeight]);
```

Comment:
this was from 2 years ago, I ran in debug + release and things look fine with it enabled. note on the exchange modal we are now using the animated component on android